exercises/atbash-cipher/src/Atbash.hs

```diff
@@ -1,7 +1,7 @@
 module Atbash (decode, encode) where
 
 decode :: String -> String
-decode phrase = error "You need to implement this function."
+decode xs = error "You need to implement this function."
```

Comment:
I don't know if it's just me, but I found `phrase` better, since `xs` doesn't really give a meaning to the string. So I think I in general didn't like `xs` as the name for any string. Even though strings are lists.

Then again, maybe the students are meant to find the meaning and rename the args to suit personal taste?

We can't please everyone so we just have to pick one, I suppose. I let y'all decide.